.github/workflows/main.yml

```diff
@@ -33,6 +33,19 @@ jobs:
       with:
         fail_ci_if_error: true
 
+  unittest-py36:
```

Comment:
You can copy paste the the above and just use ubuntu 18.04 without relying on a container.

Reply:
No, you can't do that anymore. Github complains. See the description of this PR. The only solution for supporting 3.6 is to test inside an older container.

Reply:
That was much simpler! Thanks @teojgo!